packages/docusaurus-theme-classic/src/theme/NavbarItem/DropdownNavbarItem.tsx

```diff
@@ -84,7 +84,6 @@ function DropdownNavbarItemDesktop({
     <div
       ref={dropdownRef}
       className={clsx('navbar__item', 'dropdown', 'dropdown--hoverable', {
-        'dropdown--left': position === 'left',
```

Comment:
Removed this class because it does not exist in Infima (since "left" position by default).

Reply:
Was wondering, is it even useful to have this dropdown--right class?

It does not even seem to have any effect in this context

```scss
  &--right {
    ^&__menu {
      right: 0;
    }
  }
```


We could simply filter the position at the very top directly in navbar so that no child receives it?

```js 
          {leftItems.map((item, i) => (
            <NavbarItem key={i} {...item} position={undefined}/>
          ))}
```

Reply:
There is actually a difference from of this class. But I didn't add this class initially, so I'm not sure it needs to be removed, therefore let's keep it.

| Before   | Right    |
| -------- | -------- |
| ![image](https://user-images.githubusercontent.com/4408379/126119322-9ebf598f-2c56-4a3b-9656-19204880b0db.png) | ![image](https://user-images.githubusercontent.com/4408379/126119976-100da582-ef1e-4863-a53b-22e4ab28b361.png)|

Reply:
Ah I see thanks